### PR TITLE
Update /invite, fix /me and /wall in PM

### PR DESCRIPTION
**Pull request by Morfent**

### CODE REVIEW

File: commands.js [hunk 1]
```diff
@@ -130,6 +130,8 @@ var commands = exports.commands = {
 			switch (targetCmd) {
 			case 'me':
 			case 'announce':
+			case 'wall':
+				break;
```
Comment (Zarel) [inline commands.js]: no

Comment (Zarel) [inline commands.js]: Specifically, you still need to rename `/wall` to `/announce`.

Comment (Morfent) [inline commands.js]: ``announce`` is still there, I just wanted to add `wall``.

**Comment by Morfent** [comment]
Fuck, didn't mean to close this
https://github.com/Zarel/Pokemon-Showdown/pull/1406

**Pull request closed without merging**